data_model/src/account.rs

```diff
@@ -126,13 +129,22 @@ impl Default for SignatureCheckCondition {
 }
 
 /// Builder which should be submitted in a transaction to create a new [`Account`]
-#[allow(clippy::multiple_inherent_impl)]
 #[derive(
-    Debug, Display, Clone, IdOrdEqHash, Decode, Encode, Deserialize, Serialize, IntoSchema,
+    Debug,
+    Display,
+    Clone,
+    IdOrdEqHash,
+    Decode,
+    Encode,
+    Deserialize,
+    Serialize,
+    IntoFfi,
+    TryFromReprC,
```

Comment:
they could be, I found it to look easier to read if they weren't guarded by the `ffi_api` or `ffi` feature. Would you like me to return the guard? Also, my intention is to remove the `no_std` requirement and the `ffi` feature long term in which case we'll get back to the current state

Reply:
> Would you like me to return the guard?

We have a pattern of making things private unless we need to use them. The guard is effectively making the item private, and since we're already used to reading complex `cfg_attr` directives, it makes sense to keep them. 

This is a personal preference only, so unless it's trivial to bring back I wouldn't bother.

Reply:
Well, a lot of other ffi-related code is wrapped into guards, and this derives are not anymore, that's why I asked.

I can't really say, which optnion is better:
1) Keep as is
2) Bring guards back
3) Make derives smart with guards inside generated code

Reply:
> Incidentally, how long do you reckon until we get rid of no_std?

When wasm dynamic linking is fully operational because those two heavily conflict. I would expect it removed during the next release cycle